components/fxa-client/src/lib.rs

```diff
@@ -61,6 +62,7 @@ pub struct FirefoxAccount {
     client: Arc<FxAClient>,
     state: State,
     flow_store: HashMap<String, OAuthFlow>,
+    attached_clients: Option<CachedResponse<Vec<http_client::GetAttachedClientResponse>>>,
```

Comment:
Do we want to store this in memory or persist it?
- If in memory is fine: we should rename this to `attached_clients_cache` to make this clear.
- If we want to persist this: this needs to be moved in `V2State`

Reply:
Ryan's notes from #2625 suggesting starting with caching in memory so I'll rename that property.